plugin/plugin.go

```diff
@@ -144,87 +146,110 @@ func (p *Plugin) OnTrafficFromClient(
 	return req, nil
 }
 
-// OnTrafficFromServer is called when a response is received by GatewayD from the server.
-func (p *Plugin) OnTrafficFromServer(
-	ctx context.Context, resp *v1.Struct,
-) (*v1.Struct, error) {
-	OnTrafficFromServerCounter.Inc()
-	resp, err := postgres.HandleServerMessage(resp, p.Logger)
-	if err != nil {
-		p.Logger.Info("Failed to handle server message", "error", err)
-	}
+type UpdateCacheRequest struct {
+	serverResponse *v1.Struct
+}
```

Comment:
- I suppose we can get rid of the wrapper struct (`UpdateCacheRequest`) and use `*v1.Struct` directly. WDYT?
- Nit: move this to the top of the file, next to other structs.

Reply:
Done. My only concern is that `*v1.Struct` makes the code harder to read, I'd say we probably can use type alias to put a name for it